public/request/search.php

```diff
@@ -25,7 +25,7 @@
 } elseif ($source == 'user' || $source == 'game-compare') {
     $order = [SearchType::User];
 } else {
-    $order = [SearchType::Game, SearchType::Achievement, SearchType::User];
+    $order = [SearchType::Game, SearchType::Hub, SearchType::Achievement, SearchType::User];
```

Comment:
I was going to suggest adding Hubs to the Game case and maybe the Achievement case, but I can't figure out if they're even used any more.

I think the source prioritization has been lost to the blade conversion. If I recall correctly, it used to be that searching on a game page would only return games and searching on a user page would only return users, and this catch-all was for everything else.